### PR TITLE
Fix responsive layout

### PR DESCRIPTION
**Pull request by Alynn022**
# Description

Please check the type of change your PR introduces:
- [ ] Bugfix (change which fixes an issue)
- [ ] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

 
- Refactored so that the CSS is responsive to the mobile and desktop after adding the timer component. 
- 
 

### Type of change

Please check the relevant option.

- [x] Non-Breaking Change
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Big Breaking change (this breaks everything and I am sorry)

# Where should reviewers start?

# How Has This Been Tested?

Please check the option(s) that best describe your tests. 

- [ ] New spec files were added
- [ ] Tests were added to existing spec files
- [ ] Some tests were changed
- [x] No tests were changed
- [ ] All tests were changed (please explain what in God's name happened)

### Tests Description

- Please add further details of your tests here.

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have removed all unused code or commented why I left it
- [ ] I have deployed my changes to a test-heroku and they work
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/Components/Inspirations/Inspirations.tsx [hunk 1]
```diff
@@ -1,24 +1,56 @@
+import React, { useEffect, useState } from 'react';
 import './Inspirations.scss';
 import { InspirationImage } from './InspirationImage';
 import { InspirationText } from './InspirationText';
+import Timer from '../Timer/Timer'
+import ChooseTime from '../ChooseTime/ChooseTime'
```
Comment (philalewis) [inline src/Components/Inspirations/Inspirations.tsx:6]: Moving the Timer and ChooseTime components inside of the Inspirations component makes this easier to style

File: src/Components/WritingPage/WritingPage.tsx [hunk 2]
```diff
@@ -6,28 +6,28 @@ import Timer from '../Timer/Timer'
 import ChooseTime from '../ChooseTime/ChooseTime'
 
 const WritingPage: React.FC = () => {
-  const [ totalSeconds, setTotalSeconds ] = useState<number>(0)
-  const [ timeChosen, setTimeChosen ] = useState(false)
-  const [ showSetTimeModal, setShowSetTimeModal ] = useState(false)
+  // const [ totalSeconds, setTotalSeconds ] = useState<number>(0)
```
Comment (philalewis) [inline src/Components/WritingPage/WritingPage.tsx]: We should take these comments out.

**Review by philalewis — CHANGES_REQUESTED** [review]
We should take out the comments. If we want to move the Timer back out, it's easy enough to do.

File: src/Components/WritingPage/WritingPage.tsx (hunk 2 above)
Comment (philalewis) [reply src/Components/WritingPage/WritingPage.tsx]: I went ahead and did it.

**Review by philalewis — APPROVED** [review]

**Review by Jayquintana — APPROVED** [review]
looks good I agree with Phil It might be tough for data if we move the timer but we can talk about that tomorrow with everyone !

**Pull request merged**